Hardcore.lua

```diff
@@ -27,10 +27,13 @@ Hardcore_Settings = {
 
 --[[ Character saved variables ]]--
 Hardcore_Character = {
-	time_tracked = 0,
-	time_played = 0,
+	time_tracked = 0, 		-- seconds
+	time_played = 0, 		-- seconds
+	accumulated_time_diff = 0, 	-- seconds
+	tracked_played_perc = nil,
```

Comment:
Nitpick: Use full word in variable. Not sure what "perc" means immediately looking at this